Mathlib/RingTheory/UniqueFactorizationDomain/Defs.lean

```diff
@@ -55,18 +55,19 @@ theorem exists_irreducible_factor {a : α} (ha : ¬IsUnit a) (ha0 : a ≠ 0) :
     hs.1⟩
 
 @[elab_as_elim]
-theorem induction_on_irreducible {P : α → Prop} (a : α) (h0 : P 0) (hu : ∀ u : α, IsUnit u → P u)
-    (hi : ∀ a i : α, a ≠ 0 → Irreducible i → P a → P (i * a)) : P a :=
+theorem induction_on_irreducible {motive : α → Prop} (a : α)
```

Comment:
This change isn't like the others. What is the reason for including it here?

Reply:
I think this was @urkud's request, but probably this (and others like it) belong in a standalone PR.